From 67eadf53de566b2f13043194ace4d5ed5fdd92a1 Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Wed, 21 Nov 2018 16:30:01 +0200
Subject: [PATCH 5175/5242] MLK-20328-3: ARM64: dts: mx8mm-evk: ak4497: adjust
 the expected format

commit  3d1c95dc27566e36e6ad4a6941349e3274c54b9e from
https://source.codeaurora.org/external/imx/linux-imx.git

Default value must be prefixed by "0".

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
(cherry picked from commit 5252bd4b838eaa849fafdfa42a9e322a012a1e98)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8mm-evk-ak4497.dts   |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-ak4497.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-ak4497.dts
index 95b09f2..438856d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-ak4497.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-ak4497.dts
@@ -71,7 +71,7 @@
 	assigned-clock-parents = <&clk IMX8MM_AUDIO_PLL2_OUT>;
 	assigned-clock-rates = <0>, <22579200>;
 	fsl,sai-multi-lane;
-	fsl,dataline,dsd = <0xff 0x11>;
+	fsl,dataline,dsd = <0 0xff 0x11>;
 	dmas = <&sdma2 0 26 0>, <&sdma2 1 26 0>;
 	status = "okay";
 };
-- 
1.7.9.5

